Name: Remove Engadget Nonsense Text
Version: 1.4.0-1
Author: Jason Robitaille
Description: Removes the nonsense text that flashes during scene transitions  (MIT license)

--- .orig/media/cryptofs/apps/usr/palm/applications/com.aol.engadget/index.html
+++ /media/cryptofs/apps/usr/palm/applications/com.aol.engadget/index.html
@@ -10,10 +10,5 @@
 	<link href="stylesheets/engadget.css" media="screen" rel="stylesheet" type="text/css" />
 </head>
 <body>
-	<h2>This text verifies your application is running.</h2>
-	<p>
-	To create a fully functional Palm application create a scene and remove this text from index.html.  
-	See the documentation on Palm Applications for more information on creating applications and scenes.
-	</p>
 </body>
 </html>
